Content.Server/StationEvents/Events/CluwneBeastSpawn.cs

```diff
@@ -0,0 +1,30 @@
+using System.Linq;
+using Content.Server.StationEvents.Components;
+
+namespace Content.Server.StationEvents.Events;
+
+public sealed class CluwneBeastSpawn : StationEventSystem
+{
+    public override string Prototype => "CluwneBeastSpawn";
+
+    public override void Started()
+    {
+        base.Started();
+        var spawnLocations = EntityManager.EntityQuery<VentCritterSpawnLocationComponent>().ToList();
+        RobustRandom.Shuffle(spawnLocations);
+
+        var mod = Math.Sqrt(GetSeverityModifier());
+
+        var spawnAmount = (1);
+        Sawmill.Info($"Spawning {spawnAmount} of cluwnebeasts");
```

Comment:
`Spawning 1 of cluwnebeasts` pro
should be `Spawning {spawnAmount} cluwnebeast(s)` or do the s properly if you really want to be perfect